keyboards/al1/al1.h

```diff
@@ -0,0 +1,44 @@
+#ifndef AL1_H
+#define AL1_H
+
+#include "quantum.h"
+
+#define CONFIG_LED_IO \
+  DDRB |= (1<<7) | (1<<6); \
+  DDRD |= (1<<0) | (1<<1);
+
+
+#define KEYMAP( \
```

Comment:
Can you rename `KEYMAP` to `LAYOUT_all` in this .h file and in the keymap.c files that reference it?